aziotctl/aziotctl-common/src/config/apply.rs

```diff
@@ -500,33 +502,30 @@ pub fn set_est_auth(
         aziot_keys_common::PreloadedKeyLocation,
     >,
     aziotcs_keys: &mut aziot_keyd_config::Principal,
+    cert_name: &str,
 ) -> Option<aziot_certd_config::EstAuth> {
     auth.as_ref().map(|auth| {
         let auth_x509 = auth.x509.as_ref().map(|x509| {
+            let identity_cert_id = format!("{}-{}", super::EST_ID_ID, cert_name);
```

Comment:
Can you replace the `EST_ID_DEVICE_ID` with this too, then? So that there aren't two different implementations of this algorithm.

Reply:
I don't understand. What are you referring to by `EST_ID_DEVICE_ID`? I removed that constant and replaced it with this.

Reply:
Sorry, didn't notice the second file's change.